src/chi/templates/app-layout.scss

```diff
@@ -0,0 +1,41 @@
+@import '_global-variables';
+@import '_global-mixins';
+
+.chi-main {
+  background-color: $app-layout-background-color;
+  height: 100%;
+  margin-top: 2.5rem;
+
+  &__title {
+    align-items: center;
```

Comment:
To support action buttons in the title on mobile, we'll need to set the default flex-direction to column and align-items to flex-start. Then on larger viewports, change flex-direction to row and align-items to center.

Also, it may be worth creating a class for the button container so we can target that and apply responsive margin-top. Utility class would work too (e.g. `-mt--2 -mt-md--0`)